frontend/apps/reader/readerui.lua

```diff
@@ -559,9 +560,15 @@ function ReaderUI:showReader(file, provider)
         return
     end
 
+    -- `readerui:doShowReader` sent us here but there're some files better handled on the FM
+    provider = provider or DocumentRegistry:getProvider(file)
+    if provider.provider == "filemanagerdocument" then
+        self:showFileManager(file, FileManagerDocument.open)
```

Comment:
Will it always be `FileManagerDocument.open` ? If yes, I would rather pass a boolean `is_filemanagerdocument=true` intsead of `filemanagerdocument_handle_callback`.